server/src/main/java/fr/uga/l3miage/tp4/models/EcosSessionProgrammationStepEntity.java

```diff
@@ -0,0 +1,24 @@
+package fr.uga.l3miage.tp4.models;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class EcosSessionProgrammationStepEntity {
+    @Id
+    Long id;
+    @Column(updatable = false)
+    final LocalDateTime dateTime = LocalDateTime.now();
```

Comment:
Ben non le final n'est pas en java, sinon ça n'a plus de sens
```suggestion
    LocalDateTime dateTime;
```